docs/openxla.md

```diff
@@ -0,0 +1,25 @@
+# OpenXLA
+
+As of ***Jun 28***, PyTorch/XLA now ***pulls*** XLA from OpenXLA. 
+OpenXLA ***is an*** [open source machine learning compiler XLA for GPUs, CPUs, and ML accelerators](https://github.com/openxla/xla). 
+
+Previous to OpenXLA, PyTorch/XLA pulled XLA directly from [TensorFlow](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/compiler/xla). With our [XLA to OpenXLA migration](https://github.com/pytorch/xla/pull/5202), PyTorch/XLA now pulls XLA from [OpenXLA](https://github.com/openxla/xla).
+
+# How to use OpenXLA
+
+For [PJRT runtime](https://github.com/pytorch/xla/blob/master/docs/pjrt.md) users, there is no change with this migration. For XRT runtime users, there is a separate [XRT branch of PyTorch/XLA](https://github.com/pytorch/xla/tree/xrt) since OpenXLA doesn't support XRT, .
+
+
+# Performance
+||Training Throughput on ResNet50|Training Throughput on ResNet50 |Training Throughput on ResNet50 |
```

Comment:
Let's actually remove the first row `Training Throughput on ResNet50`. And let's add a small sentence before this, saying:
```
Below is a performance comparison for ResNet50 pre and post the migration on different TPU hardwares. 
```

Reply:
thanks, updated